lib/api/apiUtils/authorization/aclChecks.js

```diff
@@ -86,11 +86,58 @@ function isBucketAuthorized(bucket, requestType, canonicalID) {
     requestType === 'objectGet' || requestType === 'objectHead');
 }
 
+function _isPermissionGranted(permissionList, canonicalID) {
+    if (permissionList.indexOf(publicId) > -1) {
```

Comment:
Do we need multiple `if`s, can these be combined?

Reply:
this definitely could be condensed into a single return statement

Reply:
I personally prefer the separate `if` style whenever merging into a single statement would require extra parenthesis or if it would rely on operator precedence rules to avoid them. I just think it makes the code easier to follow this way. This is just my personal preference, I let you decide if you prefer to merge or not :)

Reply:
kept the as separate conditions, I find that it reads easier that way as well